app/models/partners/profile.rb

```diff
@@ -144,5 +145,22 @@ def has_at_least_one_request_setting
         errors.add(:base, "At least one request type must be set")
       end
     end
+
+    def pick_up_email_addresses
+      # pick_up_email is a string of comma-separated emails, check specs for details
+      return if pick_up_email.nil?
+
+      emails = pick_up_email.delete(" ").delete_prefix(",").delete_suffix(",").split(",")
```

Comment:
I'm wondering if this shouldn't be extracted to a helper.

Reply:
I extracted this to another model method, `split_pick_up_emails`. Open to naming suggestions :)